server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyServerInterpreter.scala

```diff
@@ -0,0 +1,144 @@
+package sttp.tapir.server.netty
+
+import scala.concurrent.Future
+import scala.util.{Success, Try}
+
+import io.netty.bootstrap.ServerBootstrap
+import io.netty.buffer.{ByteBuf, Unpooled}
+import io.netty.channel.nio.NioEventLoopGroup
+import io.netty.channel.socket.nio.NioServerSocketChannel
+import io.netty.channel._
+import io.netty.handler.codec.http._
+import sttp.model.Method
+import sttp.monad.{FutureMonad, MonadError}
+import sttp.tapir.internal.NoStreams
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.interceptor.RequestResult
+import sttp.tapir.server.interpreter.{BodyListener, ServerInterpreter}
+import sttp.tapir.{Endpoint, endpoint, query, stringBody}
+//todo
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import sttp.monad.syntax._
+class NettyBodyListener[Future[_]](implicit m: MonadError[Future]) extends BodyListener[Future, ByteBuf] {
+  override def onComplete(body: ByteBuf)(cb: Try[Unit] => Future[Unit]): Future[ByteBuf] = {
+    //todo
+    cb(Success(())).map(_ => body)
```

Comment:
there's no streaming and the body is a strict (complete) value, so for now this is the correct implementation